content/docs/concepts/multicluster-configuration/index.md

```diff
@@ -0,0 +1,101 @@
+For applications up to a certain size, all of the microservices comprising the
+application can be running on a single orchestration platform (e.g., kubernetes cluster),
+but for many reasons (e.g., scale, redundancy, etc.) most applications will eventually
+need to be distributed and have some of their services running elsewhere.
+
+Istio supports many possible topologies for distributing the services of an
+application beyond a single cluster.
+Services within the mesh can use [service entries](/docs/concepts/traffic-management/#service-entries)
+to access standalone external services or services exposed by another loosely-coupled
+service mesh, a.k.a. *mesh federation*.
+Alternatively, if all the services can be under the same administrative control,
+you can [expand the service mesh](/docs/setup/kubernetes/mesh-expansion/) to include services running
+on VMs or bare metal hosts, or you can combine the services from more than one cluster into a
+single composite service mesh, i.e., a *multicluster mesh*.
+
+## Multicluster service mesh
+
+A multicluster service mesh is defined as a single composite service mesh composed of services
+running in more than one underlying cluster with all services running
+under a single administrative control.
+A service named `foo` in cluster 1 is the same service as `foo` in cluster 2. This is
+unlike a loosely-coupled federated service mesh where two clusters may have different
+definitions of the same service, which need to be reconciled when integrating them.
+
+A multicluster mesh has the advantage that all the services look the same to clients,
+regardless of where the workloads are actually running. It's transparent
+to the application whether it's deployed in a single or multicluster mesh.
+To achieve this, all of the services need to be managed by a single logical
+control plane, although not necessarily a single physical Istio control plane.
+There are two possible deployment approaches:
+
+1. A single Istio control plane with access to and configuration for all the services in the mesh.
+
+1. Multiple synchronized Istio control planes with replicated service and routing configurations. 
+
+Even within these two topologies, there is more than one way to configure things.
+Which approach to use and how to configure it depends on the requirements of the application
+as well as the features and limitations of the underlying cloud deployment platform.
+
+### Single control plane topology
+
+This multicluster configuration uses a single Istio control plane,
+typically running on one of the clusters, whose Pilot
+manages services on both local and remote clusters and configures the Envoy sidecars
+for all of them. This approach works best in environments where all of the contributing
+clusters have VPN connectivity so every pod in the mesh is reachable from anywhere else using
+the same IP address.
+
+{{< image width="80%" ratio="36.01%"
+    link="./multicluster-with-vpn.svg"
```

Comment:
Thank you for including diagrams. However, this diagram is overloaded with text making it very difficult to understand. Consider simplifying the diagram and removing the numbered steps from the diagram and instead including them in an ordered list in the text. You can then only have the numbers appear in the diagram making it easier to understand. Evaluate if you need to include all the technical details in the diagram. Ask yourself whether or not the readers need those technical detail labels to understand the **concept** you are trying to convey. Lastly, the lack of a legend leaves readers wondering what meaning the different types of shapes and lines have. Consider adding a legend to make their meaning clear.